src/portable/windows.rs

```diff
@@ -886,8 +886,12 @@ pub fn read_jose_keys(name: &str) -> anyhow::Result<(String, String)> {
     };
 
     Ok((
-        wsl.read_text_file(data_dir.clone() + "edbjwskeys.pem")?,
-        wsl.read_text_file(data_dir + "edbjwekeys.pem")?,
+        wsl.read_text_file(data_dir.clone() + "edbjwskeys.pem")
+            .map_err(NonLocalInstance)?
```

Comment:
I think `if not data_dir.exists() { return Err(NonLocalInstance); }`. Other errors should be propagated.